consume-more-stuff/server/routes/users.js

```diff
@@ -37,6 +38,18 @@ router.post("/", (req, res) => {
   let { name, username, email, password, status_id } = req.body;
   status_id = 1;
 
+  if (!/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(email)) {
+    return res.send(
```

Comment:
This is great but you want to return some kind of status code or possibly an object like:
```
{
  status: OK,
  message: ''
}
```
or 
```
{
  status: ERROR,
  message: 'Invalide blah blah blah'
}
```
That way the frontend can easily detect the status and handle message to the user